sdk/tests/select.rs

```diff
@@ -1429,3 +1430,26 @@ async fn select_order_by_rand_large() -> Result<(), Error> {
 
 	Ok(())
 }
+
+#[tokio::test]
+async fn select_from_none() -> Result<(), Error> {
+	let sql: &str = "
+		SELECT * FROM NONE;
+		SELECT * FROM NULL;
+		SELECT 'A' FROM NONE;
+		SELECT 'A' FROM NULL;
+		SELECT '*' FROM NONE, NONE;
+		SELECT '*' FROM NULL, NULL;
+		SELECT 'A' FROM NONE, NONE;
+		SELECT 'A' FROM NULL, NULL;
+		SELECT '*' FROM [NONE, NONE];
+		SELECT '*' FROM [NULL, NULL];
```

Comment:
`*` as a string?
```suggestion
		SELECT * FROM [NONE, NONE];
		SELECT * FROM [NULL, NULL];
```

Reply:
So should we change or remove it then maybe?

Reply:
I was thinking you were added the quotes... haven't seen I did it :D